From 50bf137c01239e291b837b52e0a9be64e5be1237 Mon Sep 17 00:00:00 2001
From: arathinx <ayyappanx.rathinam@intel.com>
Date: Fri, 3 Apr 2020 13:14:58 +0800
Subject: [PATCH] PONRTSYS-7038/PONRTSYS-7041: New ACL settings for rcu reset
 registers

RCU_RST_REQ register must be writable with read-only bits 1 and 28
RCU_RST_REQ2 Register must be writable, with read-only bits 1 2 3 20 21 23 24 29 30
---
 drivers/base/tep-sec.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/base/tep-sec.c b/drivers/base/tep-sec.c
index f0b03981454b..6bd02f8d756d 100644
--- a/drivers/base/tep-sec.c
+++ b/drivers/base/tep-sec.c
@@ -100,10 +100,10 @@ struct secure_regs gsec_r[] = {
 
 /* write protected register list */
 struct secure_regs gsec_w[] = {
-	{RCU_BASE + RST_REQ, REG_ACCESS_SELECTIVE, 0, 0xEFFFFFFB},
+	{RCU_BASE + RST_REQ, REG_ACCESS_SELECTIVE, 0, 0xEFFFFFFD},
 	{RCU_BASE + RST_STAT, REG_ACCESS_PROTECTED, 0, 0},
 	{RCU_BASE + RST_STAT2, REG_ACCESS_PROTECTED, 0, 0},
-	{RCU_BASE + RST_REQ2, REG_ACCESS_SELECTIVE, 0, 0x9FCFFFF1},
+	{RCU_BASE + RST_REQ2, REG_ACCESS_SELECTIVE, 0, 0x9E4FFFF1},
 	{RCU_BASE + INTERAPTIV_WDT_RST_EN, REG_ACCESS_SELECTIVE, 0, 0xFFFFFEFF},
 	{CHIPTOP_BASE + RST_BOOT_INFO, REG_ACCESS_PROTECTED, 0, 0},
 	{CHIPTOP_BASE + PAD_POWER_SUPPLY_SEL, REG_ACCESS_PROTECTED, 0, 0},
